releasetasks/templates/firefox/release_graph.yml.tmpl

```diff
@@ -45,7 +45,6 @@ scopes:
   - docker-worker:*
   - scheduler:*
   - project:releng:signing:format:mar
-  - project:releng:signing:cert:{{ signing_class }}
```

Comment:
Hmmm, I think you need this scope to make signingworker work properly: https://github.com/mozilla-releng/signingworker/blob/8f83a8c4f76ef20e713b8ac6f4132587a04367f4/signingworker/task.py#L22-L29